### PR TITLE
Replace datavalidator with backend version

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

File: docs/datavalidator/export.json [hunk 1]
```diff
@@ -1 +0,0 @@
...fault_$_in_meeting_id":["amendment","agenda_all_items","assignment_poll","motion_block","current_list_of_speakers","projector_countdowns","mediafile","topics","list_of_speakers","projector_message","motion_poll","motion","user","assignment","poll"],"meeting_id":1,"used_as_default_$amendment_in_meeting_id":1,"used_as_default_$agenda_all_items_in_meeting_id":1,"used_as_default_$assignment_poll_in_meeting_id":1,"used_as_default_$motion_block_in_meeting_id":1,"used_as_default_$current_list_of_speakers_in_meeting_id":1,"used_as_default_$projector_countdowns_in_meeting_id":1,"used_as_default_$mediafile_in_meeting_id":1,"used_as_default_$topics_in_meeting_id":1,"used_as_default_$list_of_speakers_in_meeting_id":1,"used_as_default_$projector_message_in_meeting_id":1,"used_as_default_$motion_poll_in_meeting_id":1,"used_as_default_$motion_in_meeting_id":1,"used_as_default_$user_in_meeting_id":1,"used_as_default_$assignment_in_meeting_id":1,"used_as_default_$poll_in_meeting_id":1}],"projector_message":[],"tag":[],"agenda_item":[],"topic":[],"motion_category":[],"motion_block":[],"motion_statute_paragraph":[],"motion_comment":[],"motion_change_recommendation":[],"assignment":[],"projection":[]}
```
Comment (jsangmeister) [inline docs/datavalidator/export.json:1]: I think this file was check in by accident, or is it needed anywhere?

**Comment by jsangmeister** [comment]
And is there a reason that the data validation action is not required anymore?

**Comment by FinnStutzenstein** [comment]
Oh, double work :D

https://github.com/OpenSlides/OpenSlides/pull/6197

> And is there a reason that the data validation action is not required anymore?

I ask myself the same.... I'll do it after #6197

**Comment by jsangmeister** [comment]
Closing in favor of #6197

**Pull request closed without merging**